mage_ai/data_preparation/storage/local_storage.py

```diff
@@ -93,9 +93,12 @@ async def write_json_file_async(self, file_path: str, data) -> None:
             )
             await file.write(fcontent)
 
-    def read_parquet(self, file_path: str, **kwargs) -> pd.DataFrame:
```

Comment:
Let's not remove `**kwargs`. It'll cause errors.
The S3Storage and GCSStorage also have the `**kwargs` param https://github.com/mage-ai/mage-ai/blob/master/mage_ai/data_preparation/storage/s3_storage.py#L96